common/plugins/eu.esdihumboldt.hale.common.instance/src/eu/esdihumboldt/hale/common/instance/io/impl/AbstractGeoInstanceWriter.java

```diff
@@ -212,14 +230,17 @@ protected Geometry unifyGeometry(Geometry geom) {
 
 			switch (windingOrder) {
 
-			case COUNTERCLOCKWISE:
+			case counterClockwise:
 				unifiedGeometry = WindingOrder.unifyWindingOrder(geom, true);
 				break;
-			case CLOCKWISE:
+			case clockwise:
 				unifiedGeometry = WindingOrder.unifyWindingOrder(geom, false);
 				break;
 			default:
-				unifiedGeometry = WindingOrder.unifyWindingOrder(geom, true);
+				if (report != null) {
+					report.error(new IOMessageImpl("WindingOrder is not set", null));
```

Comment:
Actually, in that case the winding order parameter would be set, but with a enum value that is not handled here. So it would be good if the message would state that it encountered a parameter it does not know. It could also state the enum value's name.